ocp_resources/virtual_machine_instance.py

```diff
@@ -1,14 +1,18 @@
+from __future__ import annotations
 import shlex
+from typing import Any
 
 import xmltodict
 from kubernetes.dynamic.exceptions import ResourceNotFoundError
 
-from ocp_resources.utils.constants import PROTOCOL_ERROR_EXCEPTION_DICT, TIMEOUT_4MINUTES
+from ocp_resources.utils.constants import PROTOCOL_ERROR_EXCEPTION_DICT, TIMEOUT_4MINUTES, TIMEOUT_30SEC, TIMEOUT_5SEC
 from ocp_resources.node import Node
 from ocp_resources.pod import Pod
 from ocp_resources.resource import NamespacedResource
 from timeout_sampler import TimeoutExpiredError, TimeoutSampler
 
+VMI_API_REQUEST_RETRY_PARAMS = {"timeout": TIMEOUT_30SEC, "sleep_time": TIMEOUT_5SEC}
```

Comment:
used once, please use where needed.

Reply:
I understood from https://github.com/RedHatQE/openshift-python-wrapper/pull/2240#discussion_r1891325659, that I should keep separate constants for VM and VMI.

Reply:
I think Ruth mean that you should not define it as global constent, but in the function that you use it.

Reply:
oh. I will add it